### PR TITLE
Added styling for inline spoilers

### PR DESCRIPTION
**Pull request by eliduvid**

### CODE REVIEW

File: src/ui/posting/Entry.css [hunk 1]
```diff
@@ -27,3 +27,21 @@
...
+    color: white;
+    cursor: pointer;
+    font-weight: bold;
+    font-size: small;
+    padding: .2em;
+}
+
+details.spoiler[open] summary {
+    display: none;
+}
```
Comment (smelamud) [inline src/ui/posting/Entry.css:47]: Newline.

File: src/ui/posting/Entry.css [hunk 2]
```diff
@@ -27,3 +27,21 @@
 .entry .content P IMG {
     max-width: min(900px - 3.5rem, 100vw - 4rem);
 }
+
+details.spoiler {
```
Comment (smelamud) [inline src/ui/posting/Entry.css:31]: Please, write tag names in caps: `DETAILS.spoiler`.

Comment (smelamud) [reply src/ui/posting/Entry.css:31]: And better to limit the scope of the style to `.entry` or even `.entry .content`, like it is done in styles above.

File: src/util/html.js [hunk 3]
```diff
@@ -1,5 +1,5 @@
 import sanitizeHtml from 'sanitize-html';
-import { parse as parseEmojis } from 'twemoji-parser';
+import {parse as parseEmojis} from 'twemoji-parser';
```
Comment (smelamud) [inline src/util/html.js:2]: With spaces were better.

File: src/util/html.js [hunk 4]
```diff
@@ -31,6 +31,9 @@ export function safePreviewHtml(html) {
             img: ["src", "srcset", "width", "height", "style"],
             a: ["href", "data-nodename"]
         },
+        allowedClasses: {
+            'details': 'spoiler'
```
Comment (smelamud) [inline src/util/html.js:35]: Please, use double quotes everywhere.

File: src/util/html.js [hunk 5]
```diff
@@ -53,6 +56,9 @@ export function safeHtml(html) {
             ...sanitizeHtml.defaults.allowedAttributes,
             img: ["src", "srcset", "width", "height", "style"],
             a: ["href", "data-nodename"]
+        },
+        allowedClasses: {
+            'details': 'spoiler'
```
Comment (smelamud) [inline src/util/html.js:61]: Here too.

**Review by smelamud — CHANGES_REQUESTED** [review]

File: src/util/html.js (hunk 3 above)
Comment (eliduvid) [reply src/util/html.js:2]: I should fix my IDE settings :-)

**Review by smelamud — APPROVED** [review]

**Pull request merged**